test_conformance/printf/util_printf.cpp

```diff
@@ -593,23 +741,27 @@ std::vector<printDataGenParameters> printVectorGenParameters = {
 
     //(Minimum)Two-wide,two positions after decimal
 
-    {NULL,"(1.0f,2.0f,3.0f,4.0f)","%2.2","hlf","float","4"},
+    { NULL, "(1.0h,2.0h,3.0h,4.0h)", "%2.2", "hlf", "half", "4" },
```

Comment:
Could we put this new test in the list at index 5 so that it corresponds to the new `vector_5` test that's been added, rather than changing what the `vector_0` -> `vector_4` tests do.

This test also fails on Codeplay's half implementation (it's the only new test which does) but I need to look into if that's an implementation issue or test issue.

Reply:
Done